recipe/meta.yaml

```diff
@@ -1,13 +1,16 @@
 {% set name = "mlflow" %}
-{% set version = "3.1.0" %}
+{% set version = "3.1.1" %}
 
 package:
   name: mlflow-split
   version: {{ version }}
 
 source:
-  url: https://github.com/mlflow/mlflow/archive/refs/tags/v{{ version }}.zip
-  sha256: a773a7af3f7f9f5a494a09b24902eea5f8310972eb8b6cf56474a0c4c3e32842
+  - url: https://github.com/mlflow/mlflow/archive/refs/tags/v{{ version }}.zip
+    sha256: daf73af57498ec18032a424c6a906b85313cdf764c2803ff4db2e237eff94014
+
+  - url: https://files.pythonhosted.org/packages/source/m/mlflow/mlflow-{{ version }}.tar.gz
```

Comment:
@xhochy what's the correct path that the archive file gets unpacked to ?

Reply:
It will be unpacked over the other archive.